config/locales/en.yml

```diff
@@ -165,6 +165,13 @@ en:
           inset_text_paragraph_one: Breaking the conditions of a caution usually leads to prosecution.
           inset_text_paragraph_two: If you were convicted of an offence after breaking the conditions of your caution, you’ll need to start again and select conviction instead of caution.
 
+  warning:
+    reset_session:
+      page_title: Application in progress
```

Comment:
Instead of `application` better to call it `check` or `disclosure check` or something similar.
There are a few places that need to be changed.